File: C/OJ/Leetcode/2020-4/fib.c
int iter(int a, int b, int N)
{
    if (N == 0)
    {
        return b;
    }
    else
        return iter(b, a + b, N - 1);
}

int fib(int N)
{
    if (N < 2)
        return N;
    else
        return iter(1, 1, N - 2);
}

File: R/20_03/quiz4/fib.R

fib <- rep(0,100)
fib[1] <- 0
fib[2] <- 1
for (i in 3:100){
  fib[i] <- fib[i-1]+fib[i-2]
}

write.table(fib,"user/R/output/Fibonacci_100.txt",sep="\n",row.names = FALSE,col.names = FALSE)


